ginivision/src/doc/source/supporting-tablets.rst

```diff
@@ -1,65 +0,0 @@
-Supporting Tablets
-==================
-
-The Gini Vision Library can be used on tablets, too. Some UI elements have been adapted to offer the best user experience for tablet users. Requirements and resources were also adapted.
-
-You may skip to the `Quick Checklist`_ to get an overview of the steps required for supporting tablets.
-
-Extraction Quality Considerations
----------------------------------
-
-We recommend implementing checks on tablet hardware to ensure that devices meet the Gini Vision Libraries minimum recommended hardware specifications.
-
-Many tablets with at least 8MP cameras don't have an LED flash (like the popular Samsung Galaxy Tab S2) therefore we don't require flash for tablets. For this reason the extraction quality on those tablets might be lower compared to smartphones.
-
-Hardware Requirements
----------------------
-
-We disabled the camera flash requirement for tablets. Camera flash is not a standard feature for tablets and even some popular models like the Samsung Galaxy Tab S2 don't have an LED flash.
-
-You can view the Gini Vision Library's hardware requirements `here <http://developer.gini.net/gini-vision-lib-android/javadoc/net/gini/android/vision/requirements/RequirementId.html>`_.
```

Comment:
Can you specify in the hardware requirements that flash on is only required for tablets?

Reply:
That link points to the live version which doesn't have tablets. The new javadoc mentions that:
<img width="573" alt="screen shot 2017-10-24 at 16 40 33" src="https://user-images.githubusercontent.com/13149498/31949665-2ff66778-b8da-11e7-9adb-247f4f8f9920.png">